src/data/set/function.lean

```diff
@@ -114,6 +114,14 @@ subtype.map f h
 @[simp] lemma maps_to.coe_restrict_apply (h : maps_to f s t) (x : s) :
   (h.restrict f s t x : β) = f x := rfl
 
+/-- Given a map `f` sending `univ : set α` into `t : set β`, restrict the codomain to `t`.
+Same as `subtype.coind`.-/
+def maps_to.restrict_codom {f : α → β} {t : set β} (h : maps_to f univ t) :
```

Comment:
```suggestion
@[simp]
def maps_to.restrict_codom {f : α → β} {t : set β} (h : maps_to f univ t) :
```

Reply:
Why so? Neither `maps_to.restrict` nor `set.restrict` are and in fact I would rather not have this one either.

Reply:
If you don't mark this `simp`, then you should add a lemma like `subtype.coind_coe`. If you do make this `simp`, then that lemma applies automatically.